archivist/notebooks/Create Asset and Events.ipynb

```diff
@@ -0,0 +1,355 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f5c3a1a4",
+   "metadata": {},
+   "source": [
+    "## Create Asset and Events"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8581a0ba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an Asset with corresponding events\n",
+    "#\n",
+    "# Main function, establishes a connection to RKVST using an App Registration then uses that\n",
+    "# to create an asset with events.\n",
```

Comment:
Sorry it's a petty one, but we should be consistent with Capitalising our special nouns like Asset, Event, Access Policy etc.

So please capitalise these 2.

Reply:
```suggestion
    "# to create an Asset with Events.\n",
```